.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Test
+
+on: push
+
+jobs:
+  test:
+    name: Bootstrap app, run linters and tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [14, 16]
+    steps:
+      - name: Checkout
```

Comment:
Should we add this GitHub actions as the first step:
https://github.com/styfle/cancel-workflow-action

Then it can stop the previous run when we run another one 🤔

Reply:
Yes, it's a good idea, added in [0cb178b](https://github.com/nimblehq/react-templates/pull/72/commits/0cb178b27c030f78ea36fbad2bd74ab1ee4a6811)
and tested that the previous run is canceled when there's a newer push

![Screen Shot 2565-04-08 at 11 10 28](https://user-images.githubusercontent.com/1772999/162360998-3d7766d2-edf2-4a65-9e4b-eb938ba4419b.png)